client/src/components/Ratings.jsx

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable max-len */
+/* eslint-disable import/extensions */
+import React from 'react';
+import EachRating from './EachRating.jsx';
+
+const Ratings = ({ ratings }) => (
+  Object.keys(ratings).map((rating) => <EachRating category={rating} rating={ratings[rating]} />)
```

Comment:
On line 7, could you change 'rating={ratings[rating]}' to 'rating={rating}'?

Reply:
The destructured ratings prop is actually an object, and I just want the numerical value stored at the rating property of the object (I did try your suggestion though, just to make sure I wasn't missing something and it didn't like it haha). Good eye though! Appreciate your diligence.